src/dataMap.js

```diff
@@ -194,6 +194,12 @@ class DataMap {
    * @returns {string|number} Column property, physical column index or passed argument.
```

Comment:
```suggestion
   * @returns {number} Column physical column index or passed argument.
```
I assume that the method returns only numbers?

Reply:
It handle also strings which shows this test: https://github.com/handsontable/handsontable/blob/b267f6c428386d939fa86092f76f18d4f1f0317f/test/e2e/core/colToProp.spec.js#L38-L40

It's strange for me, but I haven't investigated that and I won't change that. Will be a description `@param {string|number} column Visual column index or another passed argument.` good enough?